client/src/components/Upload/Collection.vue

```diff
@@ -169,12 +170,20 @@ export default {
         this.initFtpPopover();
         // file upload
         this.initUploadbox({
-            url: this.app.uploadPath,
+            initUrl: (index) => {
```

Comment:
Can you put this into the mixin these components shared, because this code is identical?